scrapers/TheNude.yml

```diff
@@ -23,7 +23,8 @@ xPathScrapers:
   performerScraper:
     performer:
       Name: (//meta[@itemprop="name"])[1]/@content
-      URL: (//meta[@itemprop="url"])[1]/@content
+      URL:
+        selector: //meta[@itemprop="url"]/@content
```

Comment:
was there an issue with the previous selector?
this now fetches more than one results (you can check the stash log (set to debug) when scraping)

Reply:
On my end, the previous selector was never populating URL with this scrapper. My stash instance runs in a docker container in Unraid. Changing the XPath to your suggestion led the URL to not be provided again. What behavior are you seeing on your end with the old XPath and my change?